assets/test/ViewerApp.test.tsx

```diff
@@ -177,5 +109,5 @@ test('Shows sign out link', () => {
     ),
   );
 
-  expect(wrapper.find('#sign-out-link').props().href).toBe('/path');
+  expect(screen.getByRole('link')).toHaveAttribute('href', '/path');
```

Comment:
Since this selector is so general, this test would fail if we ever added another link on the page. We should add the link text to the query here (which will also make the test clearer to read).